andela/travel_tool_back
# #160710722 Enable travel administrator to view accommodation centres

#### What does this PR do?
Create an endpoint for a travel admin to view guesthouses

#### Description of Task to be completed?
- Create endpoint to get all guesthouses
- Write tests for created endpoints

#### How should this be manually tested?
- Clone the repository and enter the project's folder
- Checkout to this branch and pull the branch
- Assign yourself the travel admin role by changing your roleId to 29187 in the users table
- Start the application using 'make start'
- Open postman and provide your token in the authorization header. Enter url `GET 127.0.0.1:5000/api/v1/guesthouses`
- You should see the list of the guesthouses

#### Any background context you want to provide?
Before getting a guesthouse, you can first create the guesthouse by using `POST 127.0.0.1:5000/api/v1/guesthouses`. See [PR](https://github.com/andela/travel_tool_back/pull/65) for guesthouse structure

#### What are the relevant pivotal tracker stories?
[#160710722](https://www.pivotaltracker.com/story/show/160710722)

#### Screenshots (if appropriate)
<img width="1434" alt="screen shot 2018-09-27 at 11 18 19 pm" src="https://user-images.githubusercontent.com/25195857/46177708-ced40e00-c2ab-11e8-90e0-70c9362386de.png">

#### Questions:
N/A
